swagger/v1/api_spec.json

```diff
@@ -3870,7 +3871,8 @@
               "npq-executive-leadership",
               "npq-early-years-leadership",
               "npq-additional-support-offer",
-              "npq-early-headship-coaching-offer"
+              "npq-early-headship-coaching-offer",
+              "npq-leading-primary-mathematics"
```

Comment:
since we're amending the `api_spec`, we will also need to amend everywhere this appears in the component schemas, could you search everywhere for another specialist course under `swagger/` and add that value there please? This applies to all versions (example: `swagger/v1/component_schemas/NPQParticipantDeferAttributes.yml`)